frontend/components/panels/WalletMkrDepositCheckPanel.vue

```diff
@@ -0,0 +1,152 @@
+<template>
```

Comment:
May I ask why have you reused `AllowanceAmountCheckPanel.vue` component, but not `WalletVatDaiBalanceCheckPanel.vue`? Can's you also simply `currency` prop there (and possibly rename the component)?